plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -85,7 +85,7 @@ type ReferenceManager struct {
 	projectLister              gardencorelisters.ProjectLister
 	quotaLister                gardencorelisters.QuotaLister
 	controllerDeploymentLister gardencorelisters.ControllerDeploymentLister
-	exposureClassLister        gardencorev1alpha1listers.ExposureClassLister
+	exposureClassLister        gardencorev1beta1listers.ExposureClassLister
```

Comment:
Can we use the internal lister here, please? I never understood why we haven't done it this way, so let's rather strive for consistency now that we touch this code.